Backendless/Persistence/QueryOptions.cs

```diff
@@ -23,12 +23,13 @@ public QueryOptions( int pageSize, int offset, string sortBy )
       SortBy = new List<String> {sortBy};
     }
 
-    public QueryOptions( int pageSize, int offset, string sortBy, int relationsDepth )
+    public QueryOptions( int pageSize, int offset, int relationsPageSize , string sortBy, int relationsDepth )

```

Comment:
instead of modifying the existing constructor, add a new one. This will avoid breaking backward compatibility for those who already use this constructor